openmc_dagmc_wrapper/neutronics_model.py

```diff
@@ -462,7 +498,65 @@ def export_xml(
         # this is the underlying geometry container that is filled with the
         # faceted DAGMC CAD model
         dag_univ = openmc.DAGMCUniverse(self.h5m_filename)
-        geom = openmc.Geometry(root=dag_univ)
+
+        if self.reflective_angles is None:
+            # if a graveyard is not found in the dagmc geometry a CSG one is
+            # made
+            if 'graveyard' not in di.get_materials_from_h5m(self.h5m_filename):
```

Comment:
@Shimwell it seems this line is failing in the tests

Reply:
ah yes, sorry I can see what I have messed up there. I shall fix now